ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -341,7 +346,7 @@ void RSkComponentTextInput::keyEventProcessingThread(){
       inputQueue.pop();
       privateVarProtectorMutex.unlock();
       processEventKey(eventKeyType,&stopPropagation,&waitForupdateProps, false);
-      if (waitForupdateProps)
+      if (waitForupdateProps && textString.length())
```

Comment:
Why is the second condition required? 
What happens if value prop is empty string or if clear() method is been called along with value prop

Reply:
I added this because, textstring/displayString deleted completely(example case displaystring=""). 
After that we press delete/backspace it will hang. I got different Idea to resolve that, Going to update the soon.